cmake/Modules/OpmSatellites.cmake

```diff
@@ -159,7 +159,11 @@ macro (opm_data satellite target dirname)
   set (${satellite}_INPUT_FILES)
   if (NOT PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
 	foreach (input_datafile IN LISTS ${satellite}_DATA)
-	  file (RELATIVE_PATH rel_datafile "${PROJECT_SOURCE_DIR}" ${input_datafile})
+	  if (IS_ABSOLUTE ${input_datafile})
+      file (RELATIVE_PATH rel_datafile "${PROJECT_SOURCE_DIR}" ${input_datafile})
+    else()
+      set(rel_datafile ${input_datafile})
+    endif()
```

Comment:
Is there some tab-vs-space indenting here?  Just casually reading this section it's not immediately clear that the conditional is within the `foreach` loop.

Reply:
yes, it's a mess. i plan to do a reformatting pr later.

Reply:
>> Is there some tab-vs-space indenting here?
>
> yes, it's a mess. i plan to do a reformatting pr later.

Very well.  Merge at will.